nucypher/crypto/powers.py

```diff
@@ -198,6 +198,9 @@ class TLSHostingPower(KeyPairBasedPower):
     _keypair_class = HostingKeypair
     provides = ("get_deployer",)
 
+    def public_material(self):
```

Comment:
:confounded:

Reply:
It's perfectly reasonable for a Power - even a KeypairBasedPower - to return material that isn't a key or that is a superset of a key.  I think that the generic term, "public material" is a more sober and reasonable way to consider this domain.

Reply:
![image](https://user-images.githubusercontent.com/3598318/44904280-0f039900-accc-11e8-9157-33126795c9a3.png)

Reply:
#395